utils/Actions.js

```diff
@@ -1,56 +1,134 @@
 const Attachment = require('discord.js').Attachment;
 const UrlValidator = require('./UrlValidator.js');
 
-archiveMedia = (message) => {
+const DISCORD_BASE_URL = 'https://discordapp.com';
+
+const YOUTUBE_LINKS = 'youtube-links';
+const INSTAGRAM_LINKS = 'instagram-links';
+const SPOTIFY_LINKS = 'spotify-links';
+const VLIVE_LINKS = 'vlive-links';
+const TWITTER_LINKS = 'twitter-links';
+const REDDIT_LINKS = 'reddit-links';
+const GENERAL_LINKS = 'general-links';
+const MEDIA_LINKS = 'media-links';
+const MEDIA_ATTACHMENTS = 'media-attachments';
+
+const archiveMedia = (message) => {
     console.log(message);
     const guild = message.guild;
     if (!guild) return;
 
+    maybeSendYoutubeLinks(message);
+    maybeSendInstagramLinks(message);
+    maybeSendSpotifyLinks(message);
+    maybeSendVLiveLinks(message);
+    maybeSendTwitterLinks(message);
+    maybeSendRedditLinks(message);
+    maybeSendGeneralLinks(message);
+    maybeSendMediaLinks(message);
+    maybeSendAttachments(message);
```

Comment:
Not sure how I feel about going from calling `embeds.forEach()` once to calling it 8 times. If you want to refactor the code into different functions maybe perform on the embed instead of the message? Not really a big deal for a discord bot though.

```
embeds.forEach(embed => {
  maybeSendYoutubeLink(embed);
  maybeSend...
});
...
attachments.forEach(maybeSendAttachments);
```